deploy/mayastor-daemonset.yaml

```diff
@@ -80,11 +74,11 @@ spec:
           # belong to Guaranteed QoS class, hence can never get evicted in case of
           # pressure unless they exceed those limits. limits and requests must be the same.
           limits:
-            cpu: "2"
+            cpu: "1"
             memory: "512Mi"
             hugepages-2Mi: "1Gi"
           requests:
-            cpu: "2"
+            cpu: "1"
```

Comment:
Can this me {{cpus}}.len() or something?

Reply:
yup. It is more or less like you have described in template directory of the mayastor chart. This file is the result of templating process so that's why we see hardcoded value here.